CalibPPS/AlignmentGlobal/plugins/PPSAlignmentHarvester.cc

```diff
@@ -209,8 +209,8 @@ void PPSAlignmentHarvester::fillDescriptions(edm::ConfigurationDescriptions& des
   desc.add<bool>("overwrite_sh_x", true);
   desc.add<std::string>("text_results_path", "./alignment_results.txt");
   desc.add<bool>("write_sqlite_results", false);
-  desc.add<bool>("x_ali_rel_final_slope_fixed", true);
-  desc.add<bool>("y_ali_final_slope_fixed", true);
+  desc.add<bool>("x_ali_rel_final_slope_fixed", false);
+  desc.add<bool>("y_ali_final_slope_fixed", false);

```

Comment:
There are two usages of this plugin in CMSSW:
- From Configuration/StandardSequences/python/AlCaHarvesting_cff.py, where the CalibPPS/AlignmentGlobal/python/PPSAlignmentHarvester_cff.py is imported, which already sets those two parameters as "false" and therefore no change is expected because of this modified default value
- From CalibPPS/AlignmentGlobal/test/run_analysis_cfg.py, which imports the autogenerated ppsAlignmentHarvester_cfi.py instead, and therefore these two parameters would get modified in that test.

Given that, are we supposed to switch back to "true" these two parameters in CalibPPS/AlignmentGlobal/test/run_analysis_cfg.py?

Reply:
For the sake of clarity, would you mind profiting of this PR to also modify in CalibPPS/AlignmentGlobal/python/PPSAlignmentHarvester_cff.py

From
```PYTHON
ppsAlignmentHarvester = ppsAlignmentHarvester_.clone()
ppsAlignmentHarvester.text_results_path = cms.string("")
ppsAlignmentHarvester.write_sqlite_results = cms.bool(True)
ppsAlignmentHarvester.x_ali_rel_final_slope_fixed = cms.bool(False)
ppsAlignmentHarvester.y_ali_final_slope_fixed = cms.bool(False)
```
to 
```PYTHON
ppsAlignmentHarvester = ppsAlignmentHarvester_.clone( 
                            text_results_path = "",
                            write_sqlite_results = True
                        )
```